spatialprofilingtoolbox/workflow/common/sparse_matrix_puller.py

```diff
@@ -7,7 +7,7 @@
 from psycopg2.extensions import cursor as Psycopg2Cursor
 
 from spatialprofilingtoolbox.db.expressions_table_indexer import ExpressionsTableIndexer
-from spatialprofilingtoolbox.db.study_access import StudyAccess
+from spatialprofilingtoolbox.db.accessors import StudyAccess
```

Comment:
This one I think I corrected locally but did not commit, by accident. Noticing it now on re-test.